README.md

```diff
@@ -1,2 +1,78 @@
 # rust
-Docker image for Rust
+
+This is a Docker image with Rust 1.18 installed.
+
+## How to Use
+
+In order to use this image for your project, edit the `techio.yml` file and add the following lines to your project:
+
+```yaml
+    runner: techio/rust:latest
+```
```

Comment:
But I guess you don't want to build all the project each time you run a test? A build command is called once during the build step. Then the project is fully built when you run the test. Only the stubs are rebuilt. I think it will increase the time to run a test…